src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply,";
+    public static final String BUY = "buy,";
+    public static final String RESULT = "result,";
+    public static final String DELIMETER = ",";
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int buy = 0;
+    private int supply = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        readData(fromFileName);
+        extracted(supply,buy);
+        writeData(buy, supply, toFileName, stringBuilder.toString());
+    }
+
+    private void extracted(int supply, int buy) {
+        stringBuilder.append(SUPPLY)
+                    .append(supply)
+                    .append(System.lineSeparator())
+                    .append(BUY)
+                    .append(buy)
+                    .append(System.lineSeparator())
+                    .append(RESULT)
+                    .append(supply - buy)
+                    .append(System.lineSeparator());
+    }
+
+    public static boolean isFileExists(File file) {
+        return file.exists();
+    }
+
+    private void analysisData(String value) {
```

Comment:
this method should be definitely renamed as its purpose isn't clear. Also, after your fields removal (as I described in previous comments) probably this method should return some value